static/js/stats.js

```diff
@@ -41,6 +41,8 @@ function countMarkers(map) { // eslint-disable-line no-unused-vars
             }
         })
 
+        document.getElementById('stats-pkmn-label').innerHTML = 'Pokémon (' + pkmnTotal + ')'
```

Comment:
I was going to ask why not using jQuery, but then saw it was the same thing in the whole file... :man_shrugging: